### PR TITLE
issue #7671 : Updated main.go new.go

### PR DESCRIPTION
**Pull request by RickDeb2004**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by nebula-aac — CHANGES_REQUESTED** [review]
@RickDeb2004 Can you update this open PR to tell us what does this PR do?

While we can see the files changes, we want to ensure that whomever is reviewing can gain some understanding of what they are reviewing on this PR.

Also can you update your PR to fix the lint errors as well?

**Comment by Philip-21** [comment]
Hello  @RickDeb2004 would like to know what you are working

File: mesheryctl/pkg/utils/testdata/new.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    if err != nil {
+        
+    }
+
+    if err := client.Ping(ctx); err != nil {
+    
+    }
+
+    
+    if ctx.Err() == context.DeadlineExceeded {
```
Comment (Philip-21) [inline mesheryctl/pkg/utils/testdata/new.go:21]: please give some details about this

File: mesheryctl/pkg/utils/testdata/new.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        
+    }
+}
```
Comment (Philip-21) [inline mesheryctl/pkg/utils/testdata/new.go:24]: please give some details about this

**Comment by RickDeb2004** [comment]
> @RickDeb2004 Can you update this open PR to tell us what does this PR do?
> 
> While we can see the files changes, we want to ensure that whomever is reviewing can gain some understanding of what they are reviewing on this PR.
> 
> Also can you update your PR to fix the lint errors as well?

yes I will

**Comment by RickDeb2004** [comment]
review : #7746 
here I have given a brief description
@Philip-21 @nebula-aac

**Comment by abdullah1308** [comment]
@RickDeb2004 Refer https://github.com/meshery/meshery/pull/7746#issuecomment-1567373400

Closing PR due to an absence of relevant changes. Feel free to open another one

**Pull request closed without merging**